types_test.go

```diff
@@ -58,6 +58,16 @@ func TestParseFAHDuration(t *testing.T) {
 			time.Hour*24 + time.Second,
 			false,
 		},
+		{
+			"1.5 days",
+			time.Hour*36,
+			false,
+		},
+		{
+			"1.5 day 1 sec",
```

Comment:
Is this case actually possible? I don't want to add unnecessary specifications.

Reply:
I'm not sure, I don't believe the FAH client is open source. From looking at outputs from the telnet API, I don't think it's possible. Do you want to just remove the test, or would you want to remove the second change in `types.go` also?

Reply:
Thanks for removing the test; I think its best to keep the second change-it's the safest option.